armi/reactor/converters/axialExpansionChanger/axialExpansionChanger.py

```diff
@@ -94,6 +91,9 @@ class AxialExpansionChanger:
     # presumed to be valid of a warning that may warrant further investigation.
     COMP_BLOCK_HEIGHT_DIFF_THRESHOLD: float = 1e-12
 
+    # Establish the class used to redistribute mass between components.
+    MASS_REDISTRIBUTOR = RedistributeMass
```

Comment:
I'm not sure this should be ALL_CAPS.  That's usually reserved for things that are constant. It looks to me like this is meant to be drop-in replaceable for subclasses.  Or is it?

Reply:
you are correct n that it's a drop-in replacement for subclasses. 

I'm happy to change it to not be all caps. Your call!